### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Deep1Shikha**
Added gif of the game.
Made the Readme more attractive for contributors and users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
-# breakout-game
-A simple breakout game created using HTML, CSS and Javascript
+# BreakOut Game
+This game is created using HTML, CSS and Javascript
 
-![image](https://user-images.githubusercontent.com/22955675/191938991-5d3d7123-a367-4914-9e2a-4d0a8de897db.png)
+*<strong>Game Overview</strong>*
```
Comment (ppfeiler) [inline README.md:4]: Instead of html tags you could just use normal markdown. e.g. `## Game Overview`

File: README.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 
-The game would consist of 3 lives in order to clear out all the blocks.
+- The game would consist of 3 lives in order to clear out all the blocks.
+- With each fall, the number of lives gets reduced. If all the lives are lost, then the game is over.
+- The board at the bottom has to be moved horizontally, so that the ball does not touch the ground.
 
-Feel free to contribute if you have any changes or improvements ✌️
+![ezgif com-gif-maker](https://user-images.githubusercontent.com/47516502/193630932-3f4756df-afc4-4539-ad3d-d24b53fe5e21.gif)
```
Comment (ppfeiler) [inline README.md:10]: Where is this file located? It is good practice to keep the files from the readme in the project. for example in an `docs/images` folder.

**Comment by Deep1Shikha** [comment]
@DhanushNehru , please review and merge.

**Comment by DhanushNehru** [comment]
Please pull the updated code & resolve conflicts @Deep1Shikha

**Comment by Deep1Shikha** [comment]
> Please pull the updated code & resolve conflicts @Deep1Shikha

#25 created, with the updated code and resolved conflicts. @DhanushNehru , please review.

**Pull request closed without merging**